lms/djangoapps/instructor/services.py

```diff
@@ -0,0 +1,49 @@
+"""
+Implementation of "Instructor" service
+"""
+
+import logging
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey, UsageKey
+from courseware.models import StudentModule
+from instructor.views.tools import get_student_from_identifier
+import instructor.enrollment as enrollment
+
+
+log = logging.getLogger(__name__)
+
+
+class InstructorService(object):
+    """
+    Instructor service for deleting the students attempt(s) of an exam. This service has been created
+    for the edx_proctoring's dependency injection to cater for a requirement where edx_proctoring
+    needs to call into edx-platform's functions to delete the students' existing answers, grades
+    and attempt counts if there had been an earlier attempt.
+    """
+
+    def delete_student_attempt(self, student_identifier, course_id, content_id):
+        """
+        Deletes student state for a problem.
+
+        Takes some of the following query parameters
+            - content_id is a url-name of a problem
+            - unique_student_identifier is an email or username
```

Comment:
Nit: Looks like it is just student_identifier. Also, can you make them match the order in the function declaration?